src/components/com_tjcertificate/administrator/views/certificates/tmpl/default.php

```diff
@@ -25,12 +25,12 @@
 HTMLHelper::_('bootstrap.tooltip');
 HTMLHelper::_('behavior.multiselect');
 HTMLHelper::_('formbehavior.chosen', 'select');
-HTMLHelper::_('behavior.modal', 'a.modal');
+HTMLHelper::_('bootstrap.renderModal', 'a.modal');
 HTMLHelper::script('com_tjcertificate/certificateImage.min.js', $options);
 
 $listOrder = $this->escape($this->state->get('list.ordering'));
 $listDirn  = $this->escape($this->state->get('list.direction'));
-$dispatcher = JDispatcher::getInstance();
```

Comment:
It must have been used in the file. Can you check and use the alternative @shindebalu

Reply:
Done